samples/Samples.StartupHook.AspNetCoreMvc31/Properties/launchSettings.json

```
@@ -14,7 +14,10 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development",
 
-        "DOTNET_STARTUP_HOOKS": "$(ProjectDir)$(OutputPath)profiler-lib\\OpenTelemetry.Instrumentation.StartupHook.dll",
+        "DOTNET_STARTUP_HOOKS": "$(SolutionDir)bin\\tracer-home\\netcoreapp3.1\\OpenTelemetry.Instrumentation.StartupHook.dll",
+
+        "DOTNET_ADDITIONAL_DEPS": "$(SolutionDir)bin\\tracer-home\\additionalDeps",
```

Shouldn't `additionalDeps` be `AdditionalDeps` in case for Linux tests?

```suggestion
        "DOTNET_ADDITIONAL_DEPS": "$(SolutionDir)bin\\tracer-home\\AdditionalDeps",
```